node/DEVELOPER.md

```diff
@@ -30,8 +30,13 @@ sudo apt update -y
 sudo apt install -y nodejs npm git gcc pkg-config protobuf-compiler openssl libssl-dev
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 source "$HOME/.cargo/env"
+# Check the installed node version
+node -v
+# If the installed node version is below 16, see the instructions below
 ```
 
+> If node -v return version bellow 16 refer to [Installing specific node version](https://www.linode.com/docs/guides/install-nodejs-on-ubuntu-22-04/#installing-a-specific-version) or to [NVM](https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script) in order to fix it.
```

Comment:
We already have instructions how to upgrade node in line 15, and the link you're providing is only for ubuntu 22-04. I think we're better of just adding a note below the check for node version - as you did - but to refer to the instructions in line 15.

Reply:
**Suggestion:**
```
Software Dependencies

##### **Note:** Node.js Supported Version
 If your Node.js version is below the supported version specified in the client's [documentation](https://github.com/aws/glide-for-redis/blob/main/node/README.md#nodejs-supported-version), you can upgrade it using [NVM](https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script).

-   npm
-   git
-   GCC
-   pkg-config
-   protoc (protobuf compiler)
-   openssl
-   openssl-dev
-   rustup

**Dependencies installation for Ubuntu**

\```bash
sudo apt update -y
sudo apt install -y nodejs npm git gcc pkg-config protobuf-compiler openssl libssl-dev
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
source "$HOME/.cargo/env"
# Check the installed node version
node -v
\```

> **Note:** Ensure that you installed a supported Node.js version. For Ubuntu 22.04 or earlier, please refer to the instructions [here](#note:-nodejs-supported-version) to upgrade your Node.js version.